### DISCUSSION TITLE
How Stereokit interacts with Unity？

### DISCUSSION THREAD

**Discussion by jike2018** [StereoKit/StereoKit#897] [Q&A]
hello，Now I have a Stereokit APK and a Unity APK, how do they communicate with each other

**Comment by maluoi** [comment]
I'm not much of an Android programmer myself, but I've successfully used Android Intents to communicate between different Android applications. This isn't a StereoKit or Unity specific thing, but StereoKit does use real C#, which allows you to tap into C#'s official Android support.

I'd recommend checking the official docs, [`BroadcastReceiver`](https://learn.microsoft.com/en-us/dotnet/api/android.content.broadcastreceiver) and [`Context.SendBroadcast`](https://learn.microsoft.com/en-us/dotnet/api/android.content.context.sendbroadcast) will probably be great places to start.

I'm not quite so certain how you would do this in Unity, but I believe it's still doable via plugins? Or maybe some Unity Android APIs? Not too sure.
